core/server/logging/PrettyStream.js

```diff
@@ -0,0 +1,144 @@
+// jscs:disable
```

Comment:
I don't quite understand this file. It seems to do an awful lot of work, was there not already a library that could do this?